desktop/src/main/java/bisq/desktop/main/dao/governance/proposals/ProposalsView.java

```diff
@@ -554,6 +555,7 @@ private void updateViews() {
                 if (optionalMyVote.isPresent()) {
                     MyVote myVote = optionalMyVote.get();
                     Coin stake = Coin.valueOf(myVote.getBlindVote().getStake());
+                    stakeInputTextField.setValidator(new InputValidator());
```

Comment:
Shouldn't this be set at line 676 instead of
```
        stakeInputTextField.setValidator(new BsqValidator(bsqFormatter));
```

Reply:
No the validator there is fine. Just after submitting the vote the value entered is formatted with `BSQ` in the end which causes the original validator to fail. I think it is good that it is formatted with the code in this case so I didn't remove the formatting, but decided to adapt the validator.